warrior/Tools/tc_generator/tc_generator

```diff
@@ -0,0 +1,640 @@
+#!/usr/bin/env/python3
+
+import argparse
+import os
+import logging
+import site
+import sys
+from xml.etree import ElementTree as et
+
+# Argparser to add command line argumets
+parser = argparse.ArgumentParser()
+parser.add_argument('-t', '--type', choices=['cli','snmp','netconf', 'rest'],\
+ help="type argument support test cases", required=True)
+parser.add_argument('-tc','--tc_name', help="Name of the the test case ", required=True)
+parser.add_argument('--ip', help="Ip address of the device")
```

Comment:
Not Consistent with other options.
You need add -ip and --ipaddr  both - and "--" option.